base_user_role/tests/test_user_role.py

```diff
@@ -269,4 +269,6 @@ def test_group_groups_into_role(self):
         new_role = self.role_model.search([("name", "=", "Test Role")])
         self.assertTrue(new_role)
         # Check that the role has the correct groups
-        self.assertEqual(new_role.implied_ids.ids, user_group_ids)
+        self.assertEqual(
+            sorted(set(new_role.implied_ids.ids)), sorted(set(user_group_ids))
+        )
```

Comment:
I mean, please try this:

```suggestion
        self.assertEqual(set(new_role.implied_ids.ids), set(user_group_ids))
```

Reply:
It also works!